app/test/integration/launch-test.ts

```diff
@@ -24,9 +26,16 @@ describe('App', function(this: any) {
 
     const root = path.resolve(__dirname, '..', '..', '..')
 
+    const logsDir = getLogsDirectory()
+
+    console.log(`Running Spectron and logging to ${logsDir}`)
```

Comment:
@iAmWillShepherd looks like some of my unintended changes from a75a9aabafb70eac6eba7d6b1672f07889a83498 are still around. Any objections to me rebasing this PR to take them out?

Reply:
No problems at all.